src/customConfigTests/src/test/kotlin/org/vaccineimpact/orderlyweb/customConfigTests/RunWorkflowTests.kt

```diff
@@ -181,7 +184,7 @@ class RunWorkflowTests : SeleniumTest()
         createButton.click()
         wait.until(ExpectedConditions.presenceOfElementLocated(By.id("git-branch")))
 
-        assertThat(driver.findElements(By.cssSelector("#git-commit option")).count()).isEqualTo(1)
+        assertThat(driver.findElements(By.cssSelector("#git-commit option")).count()).isEqualTo(2)
```

Comment:
@EmmaLRussell I had to change this to get the test suite to pass. Do you think this is problematic or a legitimate change?

Reply:
This is kind of a 'one strike and you're out' test - it changes the state of the orderly container since refreshing git causes the fake git system to add a new commit (I *think* it will only do this up to a maximum of two) and that's what the test checks for. It should be passing on Buildkite, but it will fail locally if you don't restart the deps between test runs. I've just tested it locally, and that was the behaviour I was seeing. I don't think the test should need to change to pass on BK - it's probably best just to ignore all the other tests and concentrate of getting your workflow progress test working locally, and let BK do the rest.